bundles/pubsub/pubsub_admin_udp/README.md

```diff
@@ -0,0 +1,149 @@
+---
+title: PSA UDP 
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+   
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# PUBSUB-Admin UDP 
+
+---
+
+## Description
+
+The scope of this description is the UDP PubSub admin. 
+
+The UDP pubsub admin is used to transfer user data transparent via UDP unicast, UDP broadcast and UDP multicast.
+UDP support packets with a maximum of 64kB . To overcome this limit the admin uses a protocol service on top of UDP 
+which fragments the data to be send and these fragments are reassembled at the reception side.
+
+### IP Addresses Unicast
+
+To use UDP-unicast, 1 IP address is needed:
+This is the IP address that is bound to an (ethernet) interface (from the subscriber). 
+This IP address is defined with  the `PSA_IP` property.
+For example: `PSA_IP=192.168.1.0/24`.
+Note the example uses CIDR notation, the CIDR notation specifies the IP range that is used
+by the pubsub admin for searching the network interfaces.
+
+Note the port will be automatically assigned, when none is specified.
+A fixed port can be assigned for example with: `PSA_IP=192.168.1.0/24:34000`.
+
+### IP Address Mulicast
+
+To use UDP-multicast 2 IP addresses are needed:
+
+1. The multicast address (in the range 224.X.X.X - 239.X.X.X)
+2. IP address which is bound to an (ethernet) interface (from the publisher)
+
+These IP address are defined with the `PSA_IP` property, 
+with the definition multicast ip address at (ethernet) interface ip address
+For example: `PSA_IP=224.100.0.0/24@192.168.1.0/24`.
```

Comment:
I am not sure If it wise to reuse the `PSA_IP` when using multicast or broadcast. 

Because in that case the `PSA_IP` is not valid anymore for PSA_TCP, PSA_ZMQ, etc. 

Maybe it is a idea to always support:
1. first look for a PSA type specific PSA_IP config property:
<PSA_TYPE>_PSA_IP , e.g. UDP_PSA_IP, TCP_PSA_IP or ZMQ_PSA_IP
2. fallback to PSA_IP If not PSA type specific variant is configured:
PSA_IP
3. fallback to the first Ethernet interface